HHS/Head-Start-TTADP
# [TTAHUB-3506] fix screenshot names for consistency

## Description of change
Rename screenshots for correctness and consistency
Prepend dates per refinement discussion

## How to test
Export screenshots from the QA dashboard and confirm the widget titles match up with the filename (may be case differences) and that today's date is stuck in the front

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3506


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
